frontend/projects/upgrade/src/app/shared/services/common-dialog.service.ts

```diff
@@ -179,7 +230,7 @@ export class DialogService {
     return this.dialog.open(ImportFeatureFlagModalComponent, config);
   }
 
-  openSimpleCommonConfirmationModal(
+  CommonSimpleConfirmationModalComponent(
```

Comment:
not sure what happened here, maybe a weird merge thing, but this should be `openSimpleCommonConfirmationModal` not `CommonSimpleConfirmationModalComponent`